File: OOP/Week 07/Task/dvd.py
from item import Item

class DVD(Item):
    
    def __init__(self, title, director, playing_time):
        self.director = director
        super().__init__(title, playing_time)

    def get_director(self):
        return self.director
        
    def set_director(self, director):
        self.director = director

    def set_owned(self, owned):
        super().set_owned(owned)

    def get_owned(self):
        return super().get_owned()

    def info(self):
        print("Director:", self.director)
        super().info()

dvd = DVD("Title", "Director", 100)
dvd.info()

File: OOP/Java OOP/Week 09/Task02/DVD.java
package Task02;

public class DVD extends Item {


    private String director;

    DVD(String title, String director, String playingTime) {
        super(title, playingTime);
        this.director = director;
    }

    public String getDirector() {
        return this.director;
    }

    public void setDirector(String director) {
        this.director = director;
    }

    public boolean getOwned() {
        return super.getOwned();
    }

    public void setOwned(boolean owned) {
        super.setOwned(owned);
    }

    public void info() {
        System.out.println("Director: " + this.director);
        super.info();
    }

}